Cargo.toml

```diff
@@ -57,3 +54,9 @@ features = ["ansi", "chrono", "env-filter"]
 lto = true
 panic = 'abort'
 codegen-units = 1
+
+[patch.crates-io]
+ed25519-zebra = { git = "https://github.com/ZcashFoundation/ed25519-zebra.git", rev = "d3512400227a362d08367088ffaa9bd4142a69c7" }
+orchard = { git = "https://github.com/zcash/orchard.git", rev = "cd1e72bbcd9f2e873408aa365537c89824cb7430" }
```

Comment:
This patch exists in order to get the dependency tree pinned correctly. We don't actually depend on the `orchard` crate yet, hence no change to `contrib/debian/copyright` re: BOSL.